.github/actions/nf-test-action/action.yml

```diff
@@ -38,27 +38,20 @@ runs:
         version: "${{ env.NFT_VER }}"
         install-pdiff: true
 
-    - name: Install dependencies from universe repo
-      if: contains(inputs.profile, 'singularity')
-      shell: bash
-      run: |
-        sudo add-apt-repository universe
-        sudo apt-get update
-        sudo apt-get install -y uidmap squashfs-tools
-
-    - name: Setup apptainer
-      if: contains(inputs.profile, 'singularity')
-      uses: eWaterCycle/setup-apptainer@3f706d898c9db585b1d741b4692e66755f3a1b40 #v2
```

Comment:
why removing singularity?

Reply:
I think we still need this for the `4cpu-linux-x64` runner

Reply:
I meant installing `uidmap` and `squashfs-tools` and the `Setup apptainer` step

Reply:
I think it should work now without it.